graphene_django/rest_framework/serializer_converter.py

```diff
@@ -72,49 +77,56 @@ def convert_serializer_to_input_type(serializer_class):
 
 
 @get_graphene_type_from_serializer_field.register(serializers.Field)
-def convert_serializer_field_to_string(field):
+def convert_serializer_field_to_string(field, **kwargs):
     return graphene.String
 
 
 @get_graphene_type_from_serializer_field.register(serializers.ModelSerializer)
-def convert_serializer_to_field(field):
+def convert_serializer_to_field(field, **kwargs):
+    return graphene.Field
+
+
+@get_graphene_type_from_serializer_field.register(serializers.PrimaryKeyRelatedField)
+def convert_serializer_key_to_field(field, is_input=True):
+    if is_input:
+        return graphene.String
     return graphene.Field
 
 
 @get_graphene_type_from_serializer_field.register(serializers.ListSerializer)
-def convert_list_serializer_to_field(field):
+def convert_list_serializer_to_field(field, **kwargs):
```

Comment:
Why start accepting `**kwargs` in these functions?

Reply:
Hey @charlex, thanks for reviewing!

IIUC, for any `convert_x_serializer_to_y`, we run these converters on both inputs and resolvers. However, we need to treat inputs differently than resolvers for certain fields like `PrimaryKeyRelatedField`. In a mutation, an InputType for a `PrimaryKeyRelatedField` should be an `ID` (`String`), but should resolve the entire `Field` object.

In `convert_serializer_field`, we immediately check for the graphene type from the serializer field, but we need to know whether or not the field is an input for `PrimaryKeyRelatedField`. So now `convert_serializer_key_to_field` needs a second argument about whether we are converting for an input or a resolver. 

We could conditionally pass extra args into `get_graphene_type_from_serializer_field` e.g.:

```py
if isinstance(field, serializers.PrimaryKeyRelatedField):
  get_graphene_type_from_serializer_field(field, is_input=is_input)
else:
  get_graphene_type_from_serializer_field(field)
```
but, I thought it would make sense to let the other `convert_x_serializer_to_y` converters flexibly accept kwargs:

```py
get_graphene_type_from_serializer_field(
  field,
  is_input=is_input,
  other_arg=something_else
)

...

convert_x_serializer_to_y(field, **kwargs):
   # conversion logic here
```

If we're cool with this approach, I'd like to also apply this logic to 1:N mutations -- `PrimaryKeyRelatedField(many=True)`

Happy to adjust the code if there's a better approach!